docs/docsite/rst/fork_clone_branch_collection.rst

```diff
@@ -0,0 +1,82 @@
+
+**Step2: Clone the forked repository**  
+
+Clone allows to copy a repository to the local machine.
+
+* Clone the forked repository in the terminal using the following command:
+
+.. code-block:: Blocks
+
+  git clone https://github.com/<Forked Organization>/ansible-aci.git
+
+Verify the name of the Git remote of your forked repository by running the following commands in the terminal:
+
+.. code-block:: Blocks
+
+  cd ansible-aci
+  git remote -v
+
+Expected output:
+
+.. code-block:: Blocks
+
+origin        https://github.com/<Forked Organization>/ansible-aci.git (fetch)
+origin        https://github.com/<Forked Organization>/ansible-aci.git (push)
+
+**Naming Convention**
+  "origin" is the default name for the first Git remote of a cloned repository. In this case, it represents your forked repo where you are going to make changes, commit, and push your code to GitHub.
+
+* Add the upstream repo as a new Git remote:
+
+  To be able to retrieve the latest changes made to the upstream project repo (CiscoDevNet/ansible-aci), we need to add it as a second Git remote. We recommend calling this second remote "upstream" and we will keep referring to it as upstream in the rest of the document.
+  Add the original CiscoDevNet repository (CiscoDevNet/ansible-aci) as a second remote named `upstream`, which will allow the fetch, and sync the latest changes:
+
+.. code-block:: Blocks
+
+  git remote add upstream https://github.com/CiscoDevNet/ansible-aci.git
+
+Adding the remote branch "upstream" is a one-time operation.
+After  adding the upstream remote,  update the local repository with the latest changes from the upstream repository:
+
+* Fetch and update the local `master` branch from upstream:
+
+.. code-block:: Blocks
+
+  git checkout master
+  git pull upstream master
+
+**Step 3: Create a Feature Branch**
+
+Branch facilitates bug fixes, addition of new features, and the integration of new versions after isolated testing. Master is the default branch of the local repository.
+Each time changes are required for a module or a new module is to be created, it is recommended that a new dedicated branch be created from master. This provides a clean branch of the latest master, enabling all necessary modifications to be made. 
+
+* Create a branch from master by using the following commands on the terminal:
+
+.. code-block:: Blocks
+
+  git checkout master
+  git checkout -b <new-branch-name>
+  git branch
+
+Maintaining changes in a dedicated branch allows the master branch to remain clean and synchronized with the upstream master. This simplifies keeping the local master branch updated without the need to merge code or rebase the master branch.
+
+**Best Practice:** Never commit directly to `master`. Use feature branches for all development work to simplify merging, testing, and collaboration.
+
+.. _fork_clone_branch_collection:
```

Comment:
what's the purpose of this at the end of the file?
.. _fork_clone_branch_collection: